environment

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Since you already use the `FOO="${FOO:-default value}"` notation, you can lave the setting of these fields inside the script